src/OrchardCore.Themes/TheAdmin/Assets/js/TheAdmin/TheAdmin.ts

```diff
@@ -197,3 +209,13 @@ $('body').on('submit', 'form.no-multisubmit', function (e) {
         form.removeClass(submittingClass);
     }, 5000);
 });
+
+declare global {
+    interface Window {
+        confirmDialog: typeof confirmDialog;
+    }
+}
+
+window.confirmDialog = confirmDialog;
```

Comment:
This is how you expose a javascript method to the global scope as it was before. Don't have time to migrate this one as it is used pretty much everywhere. Good patch for now.